views/default/admin/plugins.js

```diff
@@ -20,30 +20,13 @@ define(['jquery', 'elgg/spinner', 'elgg/Ajax', 'elgg/system_messages', 'elgg/sec
 	};
 
 	function toggleSinglePlugin(event) {
-		freezePlugins();
-
 		event.preventDefault();
 
-		var ajax = new Ajax();
+		spinner.start();
+
+		freezePlugins();
 		
-		ajax.action(this.href)
-			.done(function() {
-				// second request because views list must be rebuilt and this can't be done
-				// within the first.
-				ajax.path('admin_plugins_refresh')
-					.done(function(output) {
-
-						$('#elgg-plugin-list').html(output.list);
-						$('.elgg-sidebar').html(output.sidebar);
-
-						// reapply category filtering
-						$(".elgg-admin-plugins-categories > li.elgg-state-selected > a").trigger('click');
-						initPluginReordering();
-						unfreezePlugins();
-					});
-			}).fail(function() {
-				location.reload();
-			});
+		elgg.forward(this.href);
```

Comment:
why? 

If the event isn't prevented this is not needed, right....

Reply:
totally right